qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -0,0 +1,145 @@
+import os
+import re
+import sys
+import shutil
+
+
+# Usage of win_driver_install.py.
+def usage(script_name):
+    print "Usage of %s:" % script_name
+    use_str = " %s" % script_name
+    use_str += " --op={install_driver|uninstall_driver|query_driver|help}"
+    use_str += " --driver_path= --driver_name="
+    use_str += " --device_name= --vol_utils="
+    print use_str
+
+
+# Get dpinst.exe
+def getdpinst(vol_utils):
+    get_proc_arch_cmd = "echo %PROCESSOR_ARCHITECTURE%"
+    proc_arch = os.popen(get_proc_arch_cmd).read()
+    if "AMD64" in proc_arch:
+        dpinstdir = "%s\dpinst_64.exe" % vol_utils
+    else:
+        dpinstdir = "%s\dpinst_32.exe" % vol_utils
+    if not os.path.exists("C:\dpinst.exe"):
+        shutil.copy(dpinstdir, "C:\dpinst.exe")
+    else:
+        print "dpinst.exe is already existed"
+
+
+# Install certificate
+def certutil(vol_utils):
+    certutil_cmd = "certutil -addstore -f TrustedPublisher %s\redhat.cer" % vol_utils
+    if not os.path.exists("C:\Windows\system32\certutil.exe"):
+        shutil.copy("%s\certutil.exe" % vol_utils, "C:\Windows\system32")
+    if not os.path.exists("C:\Windows\system32\certadm.dll"):
+        shutil.copy("%s\certadm.dll" % vol_utils, "C:\Windows\system32")
+    try:
+        os.system(certutil_cmd)
+    except Exception, err:
+        print "Can not install the certificate with cmd %s!" % certutil_cmd
+        print "Details refers: %s" % err
+        exit_status = -1

```

Comment:
1 win_driver_install.py|43 col 1 error| W0612 local variable 'exit_status' is assigned to but never used [pyflakes]                                                 
  2 win_driver_install.py|56 col 1 error| W0612 local variable 'exit_status' is assigned to but never used [pyflakes]
  3 win_driver_install.py|73 col 1 error| E0602 undefined name 'install_driver_cmd' [pyflakes]
  4 win_driver_install.py|75 col 1 error| W0612 local variable 'exit_status' is assigned to but never used [pyflakes]
  5 win_driver_install.py|109 col 1 error| W0612 local variable 'exit_status' is assigned to but never used [pyflakes]